tests/aws/services/lambda_/test_lambda_integration_kinesis.py

```diff
@@ -123,12 +119,10 @@ def _send_and_receive_messages():
         # this will fail in november 2286, if this code is still around by then, read this comment and update to 10
         assert int(math.log10(timestamp)) == 9
 
-    # FIXME remove usage of this config value with 2.0
-    @patch.object(config, "SYNCHRONOUS_KINESIS_EVENTS", False)
+    # TODO: is this test relevant for the new provider without patching SYNCHRONOUS_KINESIS_EVENTS?
```

Comment:
❓ Is this test still relevant?
* It appears AWS-validated but breaking
* The patching of `SYNCHRONOUS_KINESIS_EVENTS` was only relevant for the old provider

Reply:
prob not, but let's defer that until later